lib/ruby_lsp/listeners/discover_tests.rb

```diff
@@ -0,0 +1,89 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Listeners
+    class DiscoverTests
+      extend T::Helpers
+      abstract!
+
+      include Requests::Support::Common
```

Comment:
Since its now in the parent, we can remove this `include` from the child classes.